device/arista/x86_64-arista_7060x6_16pe_384c/Arista-7060X6-16PE-384C-O128S2/th5-a7060x6-16pe-384c.config.bcm

```diff
@@ -34,7 +34,7 @@ bcm_device:
             l3_alpm_template: 1
             l3_alpm_hit_skip: 1
             sai_feat_tail_timestamp : 1
-            sai_mmu_default_cpu_config : 1
+            sai_mmu_custom_config : 1
```

Comment:
Could you please check this - https://github.com/sonic-net/sonic-buildimage/pull/21892